### PR TITLE
documentation for llama2 models

### LINKED ISSUE #25090: Need documentation for understanding the difference between `7B` and `7Bf`
Unclear what the tradeoffs are between using eg `7B` and `7Bf` as an argument for `model_size` in the `convert_llama_weights_to_hf.py` script:
https://github.com/huggingface/transformers/blob/f9cc333805c47665c6afee8b5867931e54abe0c6/src/transformers/models/llama/convert_llama_weights_to_hf.py#L65

### PR DESCRIPTION
**Pull request by shauray8**
# What does this PR do?
Adds some documentation for 'f' models.

Fixes #25090

## Before submitting
- [X] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).

## Who can review?
@ArthurZucker 

** I'm not sure if this is even the right way to include the documentation, but let me know if there's something more to add.

### CODE REVIEW

File: src/transformers/models/llama/convert_llama_weights_to_hf.py [hunk 1]
```diff
@@ -282,6 +282,7 @@ def main():
     parser.add_argument(
         "--model_size",
         choices=["7B", "7Bf", "13B", "13Bf", "30B", "65B", "70B", "70Bf", "tokenizer_only"],
+        help="'f' models are specific to the LLAMA2 official release.",
```
Comment (ArthurZucker) [inline src/transformers/models/llama/convert_llama_weights_to_hf.py:285]:
```suggestion
        help="'f' models correspond to the finetuned versions, and are specific to the Llama2 official release. For more details on Llama2, checkout the original repo: https://huggingface.co/meta-llama",
```

**Review by ArthurZucker — COMMENTED** [review]
Thanks for opening the PR! Left a small nit, since the conversion script is not part of the documentation anyway, I think this is a good place to put it!

**Comment by shauray8** [comment]
changes incorporated!

**Review by ArthurZucker — APPROVED** [review]
LGTM thanks for the addition

**Review by sgugger — APPROVED** [review]
Thanks!

**Pull request merged**